assets/scss/shortcodes/tabbed-pane.scss

```diff
@@ -29,6 +25,15 @@
   border-radius: 0;
   padding: 1.5rem;
 
+  > :last-child {
+    margin-bottom: 0;
+  }
```

Comment:
Removes excess bottom margin for the last child of text tabbed panes.

Reply:
LGTM